tests/run-staging/staged-streams_1.scala

```diff
@@ -681,18 +681,18 @@ object Test {
 }
 
 sealed trait Var[T] {
-  def get(using Quotes): Expr[T]
-  def update(e: Expr[T])(using Quotes): Expr[Unit]
+  def get: Expr[T]
+  def update(e: Quotes ?=> Expr[T]): Expr[Unit]
```

Comment:
This runtime check also found that this abstraction that we have used quite a few times was fundamentally broken. We put the `Quotes` in the wrong place which caused a potential change of owners after the creation of the quote.

Reply:
Might be good to think

- how to teach this subtlety to avoid such errors, or
- eliminate such subtleties from the framework (if possible)